packages/db/scripts/cmds/db-sql.js

```diff
@@ -64,29 +45,59 @@ export async function dbSqlCmd ({ reset, cargo, testing } = {}) {
   await client.query(configSql)
   await client.query(tablesSql)
 
-  if (cargo) {
-    if (testing) {
-      cargoSql = cargoSql.replace(
-        `
--- Create materialized view from cargo "aggregate_entries" table
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries;`,
-        `
-CREATE MATERIALIZED VIEW public.aggregate_entry
-AS
-SELECT *
-FROM cargo.aggregate_entries 
-WHERE cid_v1 in ('bafybeiaj5yqocsg5cxsuhtvclnh4ulmrgsmnfbhbrfxrc3u2kkh35mts4e');
-`
-      )
-    }
+  if (testing) {
+    await client.query(cargoSchema)
+  } else if (cargo) {
+    // Replace secrets in the FDW sql file
+    fdwSql = fdwSql.replace(":'DAG_CARGO_HOST'", `'${process.env.DAG_CARGO_HOST}'`)
+    fdwSql = fdwSql.replace(":'DAG_CARGO_DATABASE'", `'${process.env.DAG_CARGO_DATABASE}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_USER'", `'${process.env.DAG_CARGO_USER}'`)
+    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_PASSWORD'", `'${process.env.DAG_CARGO_PASSWORD}'`)
 
+    // Create web3_storage role
+    await client.query(`
+CREATE ROLE web3_storage LOGIN;
+    `)
```

Comment:
Use the `WEB3_STORAGE_USER` env var or default to `current_user`. 

```suggestion
    fdwSql = fdwSql.replaceAll(":'DAG_CARGO_PASSWORD'", `'${process.env.DAG_CARGO_PASSWORD}'`)
    fdwSql = fdwSql.replaceAll(":WEB3_STORAGE_USER", `${process.env.WEB3_STORAGE_USER || 'CURRENT_USER'}`)
```